yarn.lock

```diff
@@ -344,6 +344,22 @@ babel-generator@^6.26.0:
     source-map "^0.5.6"
     trim-right "^1.0.1"
 
+babel-helper-bindify-decorators@^6.24.1:
+  version "6.24.1"
+  resolved "https://schoology.jfrog.io/schoology/api/npm/npm/babel-helper-bindify-decorators/-/babel-helper-bindify-decorators-6.24.1.tgz#14c19e5f142d7b47f19a52431e52b1ccbc40a330"
```

Comment:
hey @ianmcnally! just tried to pull this and build it locally, and looks like you committed resolutions coming from schoology's local jfrog artifactory. Not a big deal, as I can always check out the changes, but I'm curious if you have a preferred solution for repos to ensure they're going to the public npm distributions (at least under the hood)? Lemmeknow!

Reply:
honestly, i can just look this up myself 😄  but answer if you feel like it.

Reply:
oh shoot, sorry, work computer's setup to point to an internal repo